Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM quay.io/prometheus/busybox:latest
 LABEL maintainer="The Thanos Authors"
 
-COPY thanos /bin/thanos
+COPY /thanos_tmp_for_docker /bin/thanos
```

Comment:
What issue does using this temporary thanos binary solve?

Reply:
The issue: 
* We accidently were using wrong binary in genflagdocs vs used by docker. This is because `make build` was building to current dir `./thanos` vs install-deps step is installing to GOBIN/thanos via `go install`

This is why I would love to be explicit (: and always use GOBIN. However either my dockerfile sucks or docker just cannot get anything outside current dir and recursive sub dirs.

Reply:
ack got it :+1: your Docker skillz are not bad, it is an (intentional) limitation of the Docker build tooling